x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -76,7 +76,7 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 			)
 			continue
 		}
-		if relay.Epoch > ctx.BlockHeight() {
+		if relay.Epoch > ctx.BlockHeight() || relay.Epoch < 0 {
```

Comment:
Can'y be `0` too, right? Maybe should change to `<=`?

Reply:
Technically it can be 0, so <= is not true